src/store/modules/order/OrderState.ts

```diff
@@ -0,0 +1,6 @@
+export default interface OrderState {
+    products: {
```

Comment:
You have defined OrderState but internally used `products`, it should be `orders`